package.json

```diff
@@ -22,8 +22,14 @@
   "dependencies": {
     "archiver": "^2.0.3",
     "bluebird": "^3.5.0",
+    "compare-versions": "^3.1.0",
     "docker-progress": "^2.6.3",
     "dockerode": "^2.5.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-react": "^7.4.0",
+    "eslint-plugin-standard": "^3.0.1",
```

Comment:
Where did all these `eslint-plugin-*` dependencies come from? 🐣 
Have they been added intentionally, or is this material for / from another PR?

Reply:
Good catch, that's an error. Removed.